jobsdb/readonly_jobsdb.go

```diff
@@ -626,12 +620,7 @@ func (jd *ReadonlyHandleT) GetJobByID(job_id, _ string) (string, error) {
 						job_latest_state.error_code, job_latest_state.error_response
 					FROM
 						%[1]s
-					LEFT JOIN
-						(SELECT job_id, job_state, attempt, exec_time, retry_time,
-						error_code, error_response FROM %[2]s WHERE id IN
-							(SELECT MAX(id) from %[2]s GROUP BY job_id))
-						AS job_latest_state
-					ON %[1]s.job_id=job_latest_state.job_id
+					LEFT JOIN "%[2]s" job_latest_state ON %[1]s.job_id=job_latest_state.job_id
```

Comment:
1. `job_latest_state` is no longer a valid alias.
2. With this change, the sql returns multiple rows (of the same job, with different job state)

Is this intentional?

Reply:
Forgot to use the new view here?

Reply:
thanks for spotting this!